lib/sources.nix

```
@@ -175,8 +175,8 @@ let
   */
   commitIdFromGitRepo =
     let readCommitFromFile = file: path:
-        let fileName       = toString path + "/" + file;
-            packedRefsName = toString path + "/packed-refs";
+        let fileName       = path + "/" + file;
```

```suggestion
        let fileName       = path + "/${file}";
```

`+` is left associative and `path + "/"` is a no-op.

---

Thanks